### DISCUSSION TITLE
Possibility to add key shortcuts to execute macros

### DISCUSSION THREAD

**Discussion by bernatagulloesbrina** [TabularEditor/TabularEditor3#1027] [Ideas]
Not critical but nice to have would be the ability to add key shortcuts to execute macros. 
This would open not just a new world to keyboard ninja's but also the possibility to execute macros with a stream deck which is just beyond cool. 

Think about it :)

(there may already exist a way to get to the macros with the keyboard, but I could not find it)

**Comment by otykier** [comment]
This is not possible today, but is on the backlog.
